doc/cygnus-common/installation_and_administration_guide/management_interface_v1.md

```diff
@@ -195,6 +208,282 @@ Response:
+                        {
+                            "originalEntityId":"entityId1",
+                            "originalEntityType":"entityType1",
+                            "newEntityId":"newEntityId1",
+                            "newEntityType":"newEntityType1",
+                            "attributeMappings":[
+                                {
+                                    "originalAttributeName":"attrName1",
+                                    "originalAttributeType":"attrType1",
+                                    "newAttributeName":"newAttrName1",
+                                    "newAttributeType":"newAttrType1"
+                                },{
+                                   ...
+                                }
+                            ]
+                        },{
+                           ...
+                        }
+                    ]
+                },{
+                   ...
+                }
+            ]
+        },{
+           ...
+        }
+    ]
+}
+```
+
+Response:
+
+```
+{
+    "success":"true",
+    "serviceMapping": [
+        {
+            "originalService":"service1",
+            "newService":"newService1",
+            "servicePathMappings": [
+                {
+                    "originalServicePath":"/servicePath1",
+                    "newServicePath":"/newServicePath1",
+                    "entityMappings": [
+                        {
+                            "originalEntityId":"entityId1",
+                            "originalEntityType":"entityType1",
+                            "newEntityId":"newEntityId1",
+                            "newEntityType":"newEntityType1",
+                            "attributeMappings":[
+                                {
+                                    "originalAttributeName":"attrName1",
+                                    "originalAttributeType":"attrType1",
+                                    "newAttributeName":"newAttrName1",
+                                    "newAttributeType":"newAttrType1"
+                                },{
+                                   ...
+                                }
+                            ]
+                        },{
+                           ...
+                        }
+                    ]
+                },{
+                   ...
+                }
+            ]
+        },{
+           ...
+        }
+    ]
+}
+```
+
+[Top](#top)
+
+###<a name="section5.4"></a>`DELETE /v1/namemappings`
+Updates already existent [name mappings](../../cygnus-ngsi/flume_extensions_catalogue/ngsi_name_mappings_interceptor.md). The deleted mappings are given within the request payload, following the Name Mappings Json format (for instance, if an already existent entity mapping has to be deleted, the service and the service path it belongs to must be present in the Json as well). All the mappings within the payload that are present in the configuration are deleted.
```

Comment:
Updates already existent -> Deletes already exitent

Reply:
fixed in 630fab5cd69f54d90815da1c1fe50fdfb8bc5c03